pydicom/charset.py

```diff
@@ -51,18 +51,72 @@
     'GBK': 'GBK',  # from DICOM correction CP1234
 }
 
+# Map Python encodings to escape sequences as defined in PS3.3 in tables
+# C.12-3 (single-byte) and C.12-4 (multi-byte character sets).
+escape_codes = {
+    'ascii': b'\x1b(B',  # used to switch to ASCII G0 code element
+    'iso8859': b'\x1b-A',
```

Comment:
Is this correct? The comment above has iso8859 as an alias for ascii, and the dict above has this as ISO_IR 6 which would make it `\x1b(B` wouldn't it?

There's a pretty good chance I've misunderstood, though

Reply:
No, you understood that correctly, I messed this up. It worked because the encodings are basically th same.